### PR TITLE
Update Solax Inverter + shelly plus i4.xml

### PR DESCRIPTION
**Pull request by Tizikadam**
Changed some names to make the instructions more clear, updated some devices, fixed issues

### CODE REVIEW

File: Solax/Solax Inverter.xml [hunk 1]
```diff
@@ -1,34 +1,23 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Templates format="2" protocolVersion="73">
-  <Template revision="1.0" id = "0409e0d1-59f4-48b4-ac6b-a4c890546c01">
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Templates format="2" protocolVersion="76">
+  <Template revision="1.0" id = "0409e0d1-59f4-48b4-ac6b-a4c890546c01">>
```
Comment (marian-blazicek) [inline Solax/Solax Inverter.xml]: >>
je asi preklep

File: Solax/Solax Inverter.xml [hunk 2]
```diff
@@ -1,34 +1,23 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Templates format="2" protocolVersion="73">
-  <Template revision="1.0" id = "0409e0d1-59f4-48b4-ac6b-a4c890546c01">
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Templates format="2" protocolVersion="76">
```
Comment (marian-blazicek) [inline Solax/Solax Inverter.xml:2]: zniz to na 73, aby to islo importovat aj v starsich apkach

File: Solax/Solax Inverter.xml [hunk 3]
```diff
@@ -43,7 +32,7 @@ if statusCode = "110"
  addwarning("Inverter in standby mode");
 end</ReadScriptPacketParser>
         <InternalPollInterval>20000</InternalPollInterval>
-        <CustomVariables>[{"Name":"ans","ValueTypeId":2,"NumericValue":0,"StringValue":"0"},{"Name":"tokenID","ValueTypeId":2,"NumericValue":0,"StringValue":"$[tokenID]"},{"Name":"sn","ValueTypeId":2,"NumericValue":0,"StringValue":"$[sn]"}]</CustomVariables>
+        <CustomVariables>[{"Name":"ans","ValueTypeId":2,"NumericValue":0,"StringValue":"0"},{"Name":"tokenID","ValueTypeId":2,"NumericValue":0,"StringValue":"0"},{"Name":"RegistrationNumber","ValueTypeId":2,"NumericValue":0,"StringValue":"0"}]</CustomVariables>
```
Comment (marian-blazicek) [inline Solax/Solax Inverter.xml:35]: RegistrationNumber by nemal byt import parameter ?

**Review by marian-blazicek — CHANGES_REQUESTED** [review]

File: Solax/Solax Inverter.xml (hunk 1 above)
Comment (marian-blazicek) [reply Solax/Solax Inverter.xml]: na konci ">>"

File: Solax/Solax Inverter.xml (hunk 3 above)
Comment (Tizikadam) [reply Solax/Solax Inverter.xml:35]: Normalne je tam premenna, vsetko frci

**Comment by Tizikadam** [comment]
Pridal som sem aj tu shelly plus i4

File: Shelly/Shelly Plus i4.xml [hunk 4]
```diff
@@ -0,0 +1,700 @@
...
+    <Model>i4 PLus</Model>
+    <Description></Description>
+    <Module>
+      <Name>Shelly Plus i4</Name>
+      <Id>-1</Id>
+      <Model>PacketParserModuleMqtt</Model>
+      <DeviceProperties>
+        <InternalPollInterval>5000</InternalPollInterval>
+        <CustomVariables>[{"Name":"ID","ValueTypeId":2,"NumericValue":0,"StringValue":"shellyplusi4-0cdc7ef6c104"}]</CustomVariables>
+        <IpAddress>192.168.1.14</IpAddress>
```
Comment (marian-blazicek) [inline Shelly/Shelly Plus i4.xml]: chybaju import parametre

**Review by marian-blazicek — APPROVED** [review]

**Comment by marian-blazicek** [comment]
Zatvaram. Pridal som to mimo PR.

**Pull request closed without merging**